xml/deploy_cephadm.xml

```diff
@@ -964,5 +1040,787 @@ ses-min4
+      <xref linkend="drive-groups-examples" />. The following example includes
+      all drives available to &ceph; as OSDs:
+     </para>
+<screen>
+default_drive_group_name:
+  target: '*'
+  data_devices:
+    all: true
+</screen>
+    </step>
+    <step>
+     <para>
+      Verify new layouts:
+     </para>
+<screen>
+&prompt.smaster;salt-run disks.list
+</screen>
+     <para>
+      This runner returns you a structure of matching disks based on your
+      &drvgrps;. If you are not happy with the result, repeat the previous
+      step.
+     </para>
+     <tip>
+      <title>Detailed Report</title>
+      <para>
+       In addition to the <command>disks.list</command> runner, there is a
+       <command>disks.report</command> runner that prints out a detailed
+       report.
+      </para>
+<screen>
+&prompt.smaster;salt-run disks.report
+</screen>
+     </tip>
+    </step>
+   </procedure>
+-->
+   <sect3 xml:id="drive-groups-specs">
+    <title>Specification</title>
+    <para>
+     The &drvgrps; specification file&mdash;for example,
+     <filename>drive_groups.yml</filename>&mdash;can take one of two basic
+     forms, depending on whether &bluestore; or &filestore; is to be used. For
+     &bluestore; setups, <filename>drive_groups.yml</filename> can be as
+     follows:
+    </para>
+<screen>
+drive_group_default_name:
+  target: *
+  data_devices:
+    drive_spec: <replaceable>DEVICE_SPECIFICATION</replaceable>
+  db_devices:
+    drive_spec: <replaceable>DEVICE_SPECIFICATION</replaceable>
+  wal_devices:
+    drive_spec: <replaceable>DEVICE_SPECIFICATION</replaceable>
+  block_wal_size: '5G'  # (optional, unit suffixes permitted)
+  block_db_size: '5G'   # (optional, unit suffixes permitted)
+  osds_per_device: 1   # number of osd daemons per device
+  format:              # 'bluestore' or 'filestore' (defaults to 'bluestore')
+  encryption:           # 'True' or 'False' (defaults to 'False')
+</screen>
+    <para>
+     For &filestore; setups, <filename>drive_groups.yml</filename> can be as
+     follows:
+    </para>
+<screen>
+drive_group_default_name:
+  target: *
+  data_devices:
+    drive_spec: <replaceable>DEVICE_SPECIFICATION</replaceable>
+  journal_devices:
+    drive_spec: <replaceable>DEVICE_SPECIFICATION</replaceable>
+  format: filestore
+  encryption: True
+</screen>
+   </sect3>
+   <sect3>
+    <title>Matching Disk Devices</title>
+    <para>
+     You can describe the specification using the following filters:
+    </para>
+    <itemizedlist>
+     <listitem>
+      <para>
+       By a disk model:
+      </para>
+<screen>
+model: <replaceable>DISK_MODEL_STRING</replaceable>
+</screen>
+     </listitem>
+     <listitem>
+      <para>
+       By a disk vendor:
+      </para>
+<screen>
+vendor: <replaceable>DISK_VENDOR_STRING</replaceable>
+</screen>
+      <tip>
+       <title>Lowercase Vendor String</title>
+       <para>
+        Always lowercase the <replaceable>DISK_VENDOR_STRING</replaceable>.
```

Comment:
I do not like "lowercase" as a verb.

How about:

"Always enter the <replaceable>DISK_VENDOR_STRING</replaceable> in lower case."